index.html

```diff
@@ -114,6 +114,33 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Abel Roy</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com/abelroy007" target="_blank">AbelRoy007</a>
+          </p>
+          <p class="about">Programmer by Passion, Developer by Skill! I use my skillset to help others enjoy learning programming as I do. I am a fast learner, currently learning Full Stack Web Development from resources available.[mentioned below] </p>
+          <div class="resources">
+            <p>3 Useful Web Development Resources</p>
+            <ul>
+              <li>
+                  <a href="https://www.freecodecamp.org/" target="_blank" title="FreeCodeCamp">
+                    FreeCodeCamp </a> - One of the best resources to learn from
+              </li>
+              <li>
+                <a href="https://www.sololearn.com/home" target="_blank" title="Sololearn">
+                    Sololearn </a> - A great place to start learning programming.
+              </li>
+              <li>
+                  <a href="#" target="_blank" title="E Book">
+                    Book on "Learning Web Design" </a> [E Book, Available on Amazon]
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`